### DISCUSSION TITLE
Timbrado Con solucion Factible

### DISCUSSION THREAD

**Discussion by dymarr** [bambucode/tfacturaelectronica#180] [General]
al hacer la pruba de timbrado en la opcion 3. Ejemplo CFDI 4.0 el pac devuelve  Error No.621 / Detalle: No se puede cargar el archivo como documento XML porque no es un documento XML válido.

**Comment by isalasg** [comment]
¿Ha habido alguna solución a este problema? Tengo el mismo

**Reply by gmontes** [reply]
eso pasa porque el archivo Res del ejemplo tiene un caracter que causa el error. yo Genere de nuevo el archivo RES y funciono el timbrado
[cfdi40.zip](https://github.com/bambucode/tfacturaelectronica/files/11369195/cfdi40.zip)
